tests/port.test.js

```diff
@@ -28,28 +28,56 @@ test('modify port', () => {
     expect(port.pos.y).toBe(15);
 });
 
-test('validate port: invalid values', () => {
+test('validate port: valid values', () => {
     let goodPort = new Port('label', 'layer', new Coord(0, 0));
-    let badPort = new Port('', '', new Coord(-100, -100));
-    let badLabelPort = new Port('', 'layer', new Coord(1, 1));
-    let badLayerPort = new Port('label', '', new Coord(1, 1));
-    let badCoordPort = new Port('label', 'layer', new Coord(-100, -100));
 
     expect(goodPort.validate()).toBe(true);
+});
+
+test('validate port: invalid values', () => {
```

Comment:
Can you specify which values are invalid here

Reply:
They have been added.